weld/ast.rs

```diff
@@ -276,6 +278,14 @@ impl Annotations {
         self.vectorize = Some(vectorize)
     }
 
+    pub fn parallelize(&self) -> &Option<bool> {
```

Comment:
i think `parallelize` is a misleading for this ... it should be something like `always_call_runtime` or `always_use_runtime`